packages/client/wallets/aa/src/api/CrossmintWalletService.ts

```diff
@@ -7,17 +7,6 @@ import { BaseCrossmintService } from "./BaseCrossmintService";
 export { EVMBlockchainIncludingTestnet } from "@crossmint/common-sdk-base";
 
 export class CrossmintWalletService extends BaseCrossmintService {
-    async createSessionKey(address: string | `0x${string}`) {
```

Comment:
Part of dropping the session key functionality from eoa wallets. Will re-add if someone is confident that it works and/or someone is using.

Reply:
If we decide to drop this, we need to mark it as optional on crossbit-main's endpoint.

Reply:
I cannot confirm it's working, but I wouldn't be comfortable dropping this _yet_
One of the initial reqs for the project was interop with Crossmint.com, at least the form of "I can transfer nfts from crossmint.com"